spec/support/vcr.rb

```diff
@@ -24,4 +24,8 @@
       request.proceed
     end
   end
+
+  config.ignore_request do |req|
+    req.uri.include?('register.fca') || req.uri.include?('algolia.net')
+  end
```

Comment:
This lines were added before we fixed the problem of mocking the API, is these lines necessary yet?